util/measures/2020/ia-measures-old.csv

```diff
@@ -0,0 +1,230 @@
+title,description,measureId,subcategoryId,weight,metricType,firstPerformanceYear
```

Comment:
Is this file needed?

Reply:
Nope, I copy the old one to a temp file so I can reference it for weird character issues, meant to delete it. Fixed.